NameGame/NameGame/UI/Game/GameViewController.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  GameViewController.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/27/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import UIKit
+import ReSwift
+
+protocol GameViewEventHandler {
+    func didGetUserChoice(_ choice: UserRecord)
+    func didShowResult()
+    func didShowScore()
+    func didPressEndGame()
+    func didFinishGame()
+}
+
+class GameViewController: UIViewController {
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet var userButtons: [UIButton]!
+    @IBOutlet weak var scoreLabel: UILabel!
+    
+    fileprivate var imageDictionary: [UserRecord: UIImage]!
```

Comment:
We should be able to remove the `imageDictionary` property from the `GameViewController`.